WalletWasabi.Gui/CommandLine/MixerCommand.cs

```diff
@@ -25,40 +25,28 @@ public MixerCommand(Daemon daemon)
 			};
 		}
 
-		public string WalletName { get; set; }
-		public string DestinationWalletName { get; set; }
+		public string? WalletName { get; set; }
+		public string? DestinationWalletName { get; set; }
 		public bool KeepMixAlive { get; set; }
 		public bool ShowHelp { get; set; }
 		public Daemon Daemon { get; }
 
 		public override async Task<int> InvokeAsync(IEnumerable<string> args)
 		{
-			var error = false;
-			try
+			Options.Parse(args);
```

Comment:
What does this command do now?

Reply:
Hm, it sets `ShowHelp` etc. That's very unintuitive.

Reply:
I agree, I will solve this in another PR.